src/api/admin.ts

```diff
@@ -12,11 +12,11 @@ const exec = util.promisify(cp.exec)
 app.post('/sslCerts', async (req, res) => {
   try {
     const serviceKeys = (getData('serviceKeys') || []).filter(
-      d => d.service === req.body.service
+      (d: any) => d.service === req.body.service
```

Comment:
should be `(d: ServiceKey)` no?

Also, could you change `getData` to `getProviders` or something? I think there was a helper function that returns the correct type.